privacyidea/models.py

```diff
@@ -1541,10 +1542,11 @@ class Policy(TimestampMethodsMixin, db.Model):
     def __init__(self, name,
                  active=True, scope="", action="", realm="", adminrealm="", adminuser="",
                  resolver="", user="", client="", time="", pinode="", priority=1,
-                 check_all_resolvers=False, conditions=None):
+                 check_all_resolvers=False, conditions=None, user_case_insensitive=True):
```

Comment:
See abouve. This should be False.

Reply:
[b7bc726](https://github.com/privacyidea/privacyidea/pull/3852/commits/b7bc726f2279a85bad77918536eb77dbabbf0523)